scripts/install-aspnet-codegenerator.sh

```diff
@@ -13,7 +13,7 @@ rm -rf artifacts
 dotnet tool uninstall -g dotnet-aspnet-codegenerator 
 cd $DEFAULT_NUPKG_PATH
 rm -rf microsoft.visualstudio.web.codegeneration
-rm -rf microsoft.visualstudio.web.codegeneration.contracts
```

Comment:
Does casing matter on these? For non windows builds? It’s lowercase in the previous file.

Reply:
i haven't tested these on mac/linux in a while bit I remember them working. Again, these are just helper scripts that I maintain. Definitely will circle back to these.